new_product/integrate_contracts/Customizing_Business_Logic_Contract.md

```diff
@@ -161,7 +161,7 @@ function unlock(bytes memory argsBs, bytes memory fromContractAddr, uint64 fromC
 ```
 
 - This function is invoked by the **CCM contract**. It deserializes the transaction data and invokes the asset contract to release the tokens to the target address.
-- `verifyHeaderAndExecuteTx()` in CCM contract determines the **legitimacy** of cross-chain transaction information and resolves the parameters of transaction data from the Poly chain transaction Merkle proof and `crossStateRoot` contained in the block header. 
-- After verification through Poly, the packed transaction data could be executed on the target chain.
+- `verifyHeaderAndExecuteTx()` in CCM contracts determine the **legitimacy** of cross-chain transaction information and resolves the parameters of transaction data from the Poly chain transaction Merkle proof and `crossStateRoot` contained in the block header. 
```

Comment:
`verifyHeaderAndExecuteTx()` in CCM contracts determine the **legitimacy** of cross-chain transaction information and resolve the parameters of transaction data from the Poly chain transaction Merkle proof and `crossStateRoot` contained in the block header.